hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SaslConstants.java

```diff
@@ -19,9 +19,13 @@
 
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.classification.InterfaceStability;
+import org.apache.hadoop.conf.Configuration;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.hadoop.fs.CommonConfigurationKeysPublic.HADOOP_SECURITY_SASL_MECHANISM_DEFAULT;
+import static org.apache.hadoop.fs.CommonConfigurationKeysPublic.HADOOP_SECURITY_SASL_MECHANISM_KEY;
+
```

Comment:
Calling these values constants is very misleading.

Something like SSLMechanismFactory,  SSLMechanismHolder or similar would be a more fitting name.